tests/playwright/utilities/terminal.js

```diff
@@ -0,0 +1,36 @@
+import { execSync } from 'node:child_process';
+import path from 'path';
+
+class Terminal {
+	/*
```

Comment:
indentation

Reply:
The indentation issue is caused because you are using Tabs and spaces together for indentation.
GitHub shows tabs and spaces in a different visual way.

See the image bellow:
In line 9 you have spaces
And in line 10 you have a tab

Try to be consistence. 
![image](https://github.com/elementor/elementor/assets/94905340/b8966012-898d-4aa4-8ff4-9293ca0e241b)

Reply:
@davseve Nice save! Thanks for spotting this.